addon/utils/_private/ember-node.ts

```diff
@@ -108,14 +117,8 @@ class EmberNodeView implements NodeView {
     // We set the new node here and pass it to the component to render it.
     // However, this will create a DOM mutation, which prosemirror will catch and use to rerender.
     // to avoid a NodeView rerendering when we already handled the state change, we pass true to `ignoreMutation` (by default).
```

Comment:
note that this comment is now changed too (the new `ignoreMutations` implementation has enough comments that this can probably just be removed)